Services/Administration/OJS.Services.Administration.Business/Implementations/SubmissionsForProcessingBusinessService.cs

```diff
@@ -31,5 +48,33 @@ public async Task ResetAllProcessingSubmissions()
                 await this.submissionsForProcessingData.ResetProcessingStatusById(submissionForProcessingId);
             }
         }
+
+        public void EnqueuePendingSubmissions()
+        {
+            var submissionsForProcessing = this.submissionsForProcessingData
+                .GetAllPending()
+                .ToList()
+                .Where(sfp => Math.Abs(sfp!.CreatedOn.Subtract(DateTime.UtcNow).TotalMinutes) >= 1);
+
+            if (!submissionsForProcessing.Any())
+            {
+                return;
+            }
+
+            var submissions = this.submissionsData
+                .GetByIds(submissionsForProcessing
+                    .Select(sp => sp!.SubmissionId)
+                    .ToList())
+                .MapCollection<SubmissionServiceModel>()
+                .ToList();
```

Comment:
I think there is no need to call ToList() twice.

Reply:
Fixed